### PR TITLE
Add simplified chinese  translations(Close this please)

### PR DESCRIPTION
**Pull request by maz-1**
Add simplified chinese  translations for this plugin.Tested on my archlinux.

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# German translations for pidgin-indicator package.
+# Copyright (C) 2014 Listed translators
+# This file is distributed under the same license as the pidgin-indicator package.
+# Christian Schendel <doppelhelix@gmail.com>, 2014.
```
Comment (philipl) [inline po/zh_CN.po]: You should update the comment at the top to match your details.

Comment (maz-1) [inline po/zh_CN.po]: Sorry for negligence. Should be fine now.

**Comment by philipl** [comment]
Looks great. But could you please squash the commits into a single commit? Then I can merge it. Thanks!

**Comment by maz-1** [comment]
Close this please.I will fork and file a pull request again as I have no idea how to merge pushed request.

**Pull request closed without merging**